packages/block-library/src/post-navigation-link/index.php

```diff
@@ -111,8 +110,8 @@ function render_block_core_post_navigation_link( $attributes, $content ) {
 	 */
 	$get_link_function = "get_{$navigation_type}_post_link";
 
-	if ( $in_same_term ) {
-		$content = $get_link_function( $format, $link, $in_same_term, '', $taxonomy );
+	if ( $taxonomy ) {
+		$content = $get_link_function( $format, $link, true, '', $taxonomy );
```

Comment:
```suggestion
	if ( ! empty( $attributes['taxonomy'] ) ) {
		$content = $get_link_function( $format, $link, true, '', $attributes['taxonomy'] );
```

This may be a matter of personal preference, but I think this way of writing is simple. This allows you to delete line 102:

```php
$taxonomy = isset( $attributes['taxonomy'] ) ? $attributes['taxonomy'] : '';
```

Reply:
I don't mind, done.